backend/Extensions/NewUserExtensions.cs

```diff
@@ -21,7 +21,7 @@ public static class NewUserExtensions
 
         public static bool IsValidPassword(this NewUserResource newUser)
         {
-            const string passwordRegex = "^(?=.*[a-z])(?=.*[A-Z])(?=.*[0-9])(?=.*[!@#\\$%\\^&\\*])(?=.{8,})";
+            const string passwordRegex = "^(?=.*[a-z])(?=.*[A-Z])(?=.*[0-9])(?=.*[!@#-_\\$%\\^&\\*\\?\\€\\,\\;\\.\\:])(?=.{8,})";
```

Comment:
the character `-` has a special meaning when it's within brackets. Having `#-_` will allow any character with an ASCII value between 35 and 95.
```suggestion
            const string passwordRegex = "^(?=.*[a-z])(?=.*[A-Z])(?=.*[0-9])(?=.*[!@#\\-_\\$%\\^&\\*\\?\\€\\,\\;\\.\\:])(?=.{8,})";
```